src/java/README.md

```diff
@@ -21,6 +21,8 @@ Installs Java, SDKMAN! (if not installed), and needed dependencies.
 | gradleVersion | Select or enter a Gradle version | string | latest |
 | installMaven | Install Maven, a management tool for Java | boolean | false |
 | mavenVersion | Select or enter a Maven version | string | latest |
+| installAnt | Install Ant, a software tool for automating software build processes | boolean | false |
```

Comment:
Thanks for being considerate, however, This file is autogenerated with https://github.com/devcontainers/action which utilizes the `options`. Can we revert this change to avoid merge conflicts?